profiles/models.py

```diff
@@ -141,3 +141,16 @@ def can_edit_grid_element(self):
         if getattr(settings, "RESTRICT_GRID_EDITORS", False):
             return self.user.has_perm("grid.change_element")
         return True
+
+
+class ExtraFields(BaseModel):
+    profile = models.ForeignKey(Profile, on_delete=models.CASCADE)
+    key = models.CharField(max_length=256)
+    value = models.CharField(max_length=256)
```

Comment:
+1 to this approach. I know it's more work, but it'll be easier to find all `value__contains="witter"` down the road for grouping groups of like-minded social accounts together.